unpackaged/gated/healthCheck/lwc/healthCheck/healthCheck.js

```diff
@@ -8,20 +10,38 @@ export default class HealthCheck extends LightningElement {
     @track totalChecks = 0;
     @track passedChecks = 0;
     @track lastRunDate = '';
+    @track isDisplayHealthCheckGroup = false;
+
+    @track healthCheckDefinitionsToDisplayList = [];
+
+    LabelReference = {
+        stgHealthCheckErrorLastRunDate
+    }
 
     handleHealthCheckRun(){
         updateHealthCheckLastRunDate()
-        .then(result => {
-            this.lastRunDate = result;
-        })
-
+            .then(result => {
+                this.lastRunDate = result;
+                this.isDisplayHealthCheckGroup = true;
+            })
+            .catch(error => {
+                // console.log('error updating last run date: ', error);
+                this.lastRunDate = this.LabelReference.stgHealthCheckErrorLastRunDate;
+            });
     }
 
     @wire(getHealthCheckViewModel)
-    healthCheckViewModel({ error, data}) {
-        if (data) {
-            this.lastRunDate = data.lastRunDate
-        } else if(error){}
+    healthCheckViewModel({error, data}){
+        if (data){
+            this.lastRunDate = data.lastRunDate;                
+            this.healthCheckDefinitionsToDisplayList = data.healthCheckDefinitionList;  
+        } else if (error){
+            console.log('error retrieving health check view model: ', error);
```

Comment:
Let's comment this out for now and address them when we land on how we're going to address error handling.
```suggestion
            //console.log('error retrieving health check view model: ', error);
```